contrib/kubespray/config/config.yaml

```diff
@@ -8,6 +8,14 @@ docker_image_tag: v1.5.0
 #                    OpenPAI Customized Settings                      #
 #######################################################################
 # enable_hived_scheduler: true
+# enable_docker_cache: false
+# docker_cache_storage_backend: "azure" # or "filesystem"
```

Comment:
Suggest to use the term `hostPath` or `localDisk` instead of `filesystem`. Because you can also consider `azure` as a `filesystem`.

Reply:
Also, suggest to use the term `azureblob` instead of `azure`. There are a lot of  file services in azure e.g. azure file, azure disk. `azureblob` is just one of them.